PWGGA/GammaConv/AliConversionPhotonCuts.cxx

```diff
@@ -3539,6 +3547,11 @@ Bool_t AliConversionPhotonCuts::SetSharedElectronCut(Int_t sharedElec) {
       fDoPhotonQualitySelectionCut = kTRUE;
       fPhotonQualityCut = 3;
       break;
+    case 5://Cat1 rejection
+      fDoSharedElecCut = kFALSE;
+      fDoPhotonQualityRejectionCut = kTRUE;
+      fPhotonQualityCut = 1;
+      break;
```

Comment:
please put the fDoPhotonQualitySelectionCut explicitly to kFALSE here and for the other cases the same for the rejection cut

Reply:
please put for the cases 0-4 the fDoPhotonQualityRejectionCut = kFALSE